qiita_ware/test/test_ebi.py

```diff
@@ -35,20 +34,16 @@
 
 class TestEBISubmission(TestCase):
     def setUp(self):
-        self.path = join(path.dirname(path.abspath(__file__)), 'test_data',
-                         'test_ebi')
-
-        ebi_test_file = partial(join, self.path)
-
-        self.sample1_fp = ebi_test_file('sample1.fastq.gz')
-        self.sample2_fp = ebi_test_file('sample2.fastq.gz')
-        self.sample3_fp = ebi_test_file('sample3.fastq.gz')
-
         self.temp_dir = gettempdir()
         self.files_to_remove = []
 
+        if not isdir(self.temp_dir):
```

Comment:
Why you have to specifically test here if such directory exists? Also I think that using `gettempdir()` in line 37 is not correct, what you actually want to use is [`tempfile.mkdtemp`](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp)